src/pages/BooksDetail.jsx

```diff
@@ -69,6 +80,38 @@ function BooksDetail({ match }) {
     history.push("/books");
     dispatch(changeDropdown(genre));
   }
+  console.log(favBooks.map((book) => book.id).includes(Number(paramID)));
```

Comment:
I think you need to remove these because these are new console logs which you haven't deleted them in the other PR

Reply:
done